src/LSST/system/ComConnection.cpp

```diff
@@ -242,6 +235,38 @@ void ComConnection::_sendWelcomeMsg() {
                        "Configurable_File_Description_PLACEHOLDER_surrogate_handling.csv"};
         _syncWrite(to_string(js));
     }
+
+    // FUTURE: This is only for backward compatibility and will not be needed if the final version
+    // PLACEHOLDER
+    {
+        json js;
+        js["id"] = "summaryState";
+        globals.setSummaryState(3);
```

Comment:
Same question of `3` as above.

Reply:
The GUI was failing to get into "Local" mode without a clear explanation of why it was just hanging with all the buttons grayed out. So, I added entries to the welcome message that matched exactly what the simulator in the GUI was doing until the GUI would at least go into Local mode. I need to go into the python simulator code and find out where these numbers are coming from, as I'm just logging the json messages in the GUI and com code.

Just using an enum isn't what is needed. There should be a real "summaryState" that indicates a real thing in M2 system state. That is the value that should be written here, but I'm certain what that is yet.

Reply:
To get all the received JSON command acknowledgement and event packets in logging, you can use the `DEBUG` logging level.

For the button in GUI, the related code is here:
https://github.com/lsst-ts/ts_m2gui/blob/develop/python/lsst/ts/m2gui/layout/layout_control.py#L64-L69

It is waiting for the signal here:
https://github.com/lsst-ts/ts_m2gui/blob/develop/python/lsst/ts/m2gui/model.py#L688-L694